rss2email/error.py

```diff
@@ -43,13 +43,15 @@ def log(self):
 class TimeoutError (RSS2EmailError):
     def __init__(self, time_limited_function, message=None):
         if message is None:
-            if time_limited_function.error is not None:
-                message = (
-                    'error while running time limited function: {}'.format(
-                        time_limited_function.error[1]))
-            else:
-                message = '{} second timeout exceeded'.format(
-                    time_limited_function.timeout)
+            message = ''
+        else:
+            message += ': '
+        if time_limited_function.error is not None:
+            message += 'error while running time limited function: {}'.format(
+                time_limited_function.error[1])
+        else:
+            message += '{} second timeout exceeded'.format(
+                time_limited_function.timeout)
```

Comment:
I’m not convinced pulling the logic out of the `message is None` check will keep the intended semantics between the `time_limited_function` and `message` arguments.

Previously, it passed `message` through as-is if it was not `None`, now it always adds stuff.

Reply:
There is as far as I can tell only one place this is used in the entire code-base and I believe my change is correct for that use-case.